workspaces/web/src/media/sound.js

```diff
@@ -1,15 +1,17 @@
 import { Howl } from "howler"
 
 const sound = {
-    correct: new Howl({
-        src: ["sound/correct.mp3"],
-    }),
-    wrong: new Howl({
-        src: ["sound/wrong.mp3"],
-    }),
-    fanfare: new Howl({
-        src: ["sound/fanfare.mp3"],
-    }),
+    correct: () => playAudio("sound","correct"),
```

Comment:
One danger that I see here is that this way the browser will only start downloading the sound file once it should be actually played. That could lead to some extra latency when it's first played, especially on slow networks.

Before this the download of the sound file started (afaik) when the module was imported (i. e. when `new Howl` was called